api/bindings/v1alpha1/boundendpoint_types.go

```diff
@@ -114,7 +114,7 @@ type EndpointTarget struct {
 	Port int32 `json:"port"`
 
 	// Metadata is a subset of metav1.ObjectMeta that is added to the Service
-	// +kube:validation:Optional
+	// +kubebuilder:validation:Optional
```

Comment:
this resulted in no changes to the manifest because its redundant with the `omitempty`. We have 33 of these through our types but we aren't consistently using it. I could also just remove this marker from all the types if that sounds good